src/Admin/Views/Organizations/_ViewInformation.cshtml

```diff
@@ -32,6 +32,18 @@
     <dt class="col-sm-4 col-lg-3">Collections</dt>
     <dd class="col-sm-8 col-lg-9">@Model.CollectionCount</dd>
 
+    <dt class="col-sm-4 col-lg-3">Secrets</dt>
+    <dd class="col-sm-8 col-lg-9">@(Model.Secrets == -1 ? "N/A": Model.Secrets)</dd>
+
+    <dt class="col-sm-4 col-lg-3">Projects</dt>
+    <dd class="col-sm-8 col-lg-9">@(Model.Projects == -1? "N/A" : Model.Projects)</dd>
+
+    <dt class="col-sm-4 col-lg-3">Service Accounts</dt>
+    <dd class="col-sm-8 col-lg-9">@(Model.ServiceAccounts == -1 ? "N/A" : Model.ServiceAccounts)</dd>
+
+    <dt class="col-sm-4 col-lg-3">Secrets manager Seats</dt>
+            <dd class="col-sm-8 col-lg-9">@(Model.SmSeats == -1 ? "N/A" : Model.SmSeats)</dd>
+
```

Comment:
Secrets manager isn't released for self-hosted yet, so we should either hide these fields or set to N/A.

We'll need to come back and update this check when SM is released for self-hosted. (CC @Thomas-Avery )

Reply:
I think using Model.UseSecretsManager to check should solve the problem